Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -51,6 +51,7 @@
   - Cane # DeltaV
   - WhiteCane #DeltaV
   - CDDogtags # _CD
+  - CrayonMagic # Delta V, _NF Port
```

Comment:
remove

Reply:
Will be rmeoved, jsut havent pushed my latest commit